lib/engine/game/g_1889.rb

```diff
@@ -27,6 +28,26 @@ class G1889 < Base
       EBUY_PRES_SWAP = false # allow presidential swaps of other corps when ebuying
       EBUY_OTHER_VALUE = false # allow ebuying other corp trains for up to face
       HOME_TOKEN_TIMING = :operating_round
+
+      def operating_round(round_num)
+        Round::Operating.new(self, [
+          Step::Bankrupt,
+          Step::Exchange,
+          Step::DiscardTrain,
+          Step::G1889::SpecialTrack,
+          Step::BuyCompany,
+          Step::Track,
+          Step::Token,
+          Step::Route,
+          Step::Dividend,
+          Step::BuyTrain,
+          [Step::BuyCompany, blocks: true],
+        ], round_num: round_num)
+      end
+
+      def active_players
+        current_entity == company_by_id('ER') ? [@round.company_seller] : super
```

Comment:
it's not the company player?

Reply:
why does this need an override

Reply:
Not the company player, but the previous owner. ![Screenshot from 2020-08-18 21-42-51](https://user-images.githubusercontent.com/1045173/90589820-3f37f100-e19c-11ea-9d52-c617156a31a1.png)

Needs an override because normally if a company is active, the player who can take actions, and is alerted about their turn, is the owner of the company.

Reply:
like 99% of the time it's the same owner, why would you sell to someone else. sigh